packages/@aws-cdk/aws-applicationautoscaling/lib/step-scaling-action.ts

```diff
@@ -118,6 +112,24 @@ export class StepScalingAction extends cdk.Construct implements cloudwatch.IAlar
   }
 }
 
+/**
+ * Use a StepScalingAction as an Alarm Action
+ *
+ * This class is here and not in aws-cloudwatch-actions because this library
```

Comment:
For discoverability purposes and since there is basically zero logic in this class, maybe the right approach is to have a private implementation here (even inline) and then put this class there without the cyclic dependency.

Reply:
Ew! 

But okay.